src/data/metadata/MetadataD2ApiRepository.ts

```diff
@@ -64,18 +64,14 @@ export class MetadataD2ApiRepository implements MetadataRepository {
     }
 
     @cache()
-    public async listMetadata({
-        type,
-        fields = { $owner: true },
-        page,
-        pageSize,
-        order,
-        ...params
-    }: ListMetadataParams): Promise<ListMetadataResponse> {
+    public async listMetadata(listParams: ListMetadataParams): Promise<ListMetadataResponse> {
+        const { type, fields = { $owner: true }, page, pageSize, order, ...params } = listParams;
         const filter = this.buildListFilters(params);
         const { apiVersion } = this.instance;
         const options = { type, fields, filter, order, page, pageSize };
-        const { objects, pager } = await this.getListPaginated(options);
+        const { objects: baseObjects, pager } = await this.getListPaginated(options);
```

Comment:
Pager does not include `parentObjects`

Reply:
Yes, that was intended, it simplifies things and maintenance-app does the same. @adrianq, do we add it?

![image](https://user-images.githubusercontent.com/24643/93598715-62350b00-f9bd-11ea-91e4-f76b94daa9df.png)

Reply:
No, let's keep it as is for now